cmd/flags/root.go

```diff
@@ -46,7 +45,10 @@ func (r *Root) String() string {
 
 // InitFlags initialises all the flags.
 func (r *Root) InitFlags() {
-	flag.IntVar(&r.MaxDurationSeconds, "max-duration-seconds", 60, "Max duration in seconds after which warm up will stop making requests")
+	// TODO: rename this to `max-global-duration-seconds`
```

Comment:
kept the original name to avoid breaking changes, but I think we should rename this and deprecate the old flag